### ISSUE TITLE
@deprecated Use the remove() method on the event subscription returned by addEventListener().

### ISSUE THREAD

**Issue by URvesh109** [UdaySravanK/RNSwipeButton#92]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `rn-swipe-button@1.3.7` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/rn-swipe-button/src/components/SwipeButton/index.js b/node_modules/rn-swipe-button/src/components/SwipeButton/index.js
index e77cacd..d9f7df5 100644
--- a/node_modules/rn-swipe-button/src/components/SwipeButton/index.js
+++ b/node_modules/rn-swipe-button/src/components/SwipeButton/index.js
@@ -48,7 +48,7 @@ const SwipeButton = props => {
       setScreenReaderEnabled(isEnabled);
     };
     setIsUnmounting(false);
-    AccessibilityInfo.addEventListener('change', handleScreenReaderToggled);
+    const subscription = AccessibilityInfo.addEventListener('change', handleScreenReaderToggled);
 
     AccessibilityInfo.isScreenReaderEnabled().then(isEnabled => {
       if (isUnmounting) {
@@ -59,10 +59,11 @@ const SwipeButton = props => {
 
     return () => {
       setIsUnmounting(true);
-      AccessibilityInfo.removeEventListener(
-        'change',
-        handleScreenReaderToggled,
-      );
+      subscription.remove();
+      // AccessibilityInfo.removeEventListener(
+      //   'change',
+      //   handleScreenReaderToggled,
+      // );
     };
   }, [isUnmounting, screenReaderEnabled]);
 
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by rasmusviben** [comment]
Any news for this change? :)

**Comment by ElpayetSupervan** [comment]
@JakeOrel @rasmusviben @UdaySravanK I tried to make a PR but was unable to launch the example project locally to test it. Right now it only reflects changes proposed by @URvesh109, without testing on my side.

I have this error when trying to launch the example project : 
`Unable to make field private final java.lang.String java.io.File.path accessible: module java.base does not "opens java.io" to unnamed module @3cf320c0`

when executing this command :
`yarn run android` after `yarn start`

I have installed the dependencies with yarn, setup my IDE to run JDK15. I've been able to build the project (gradle build) but not launching it.
